src/app/pages/products/products.component.ts

```diff
@@ -10,8 +11,23 @@ import {PRODUCTS} from './constants';
 export class ProductsComponent implements OnInit {
   title = 'All products';
   products: ProductInterface[] = [];
+  isLoading = true;
```

Comment:
By default it should be false, you set it to true as soon as you get the products